torch/_subclasses/fake_tensor.py

```diff
@@ -1271,6 +1271,7 @@ def cpp_meta_supports_symint(self, func):
         if torch.Tag.view_copy in func.tags:  # type: ignore[attr-defined]
             return True
         return func in [
+            aten.empty.memory_format,
```

Comment:
might be worth a quick comment in the code? "we want empty.memory_format to go through C++ instead of through prim decompositions, since it results in fewer unnecessary guards"